### PR TITLE
fix failing form e2e test due to multiple submit buttons present on page

### PR DESCRIPTION
**Pull request by cstns**
## Description

fixes constantly failing form test due to multiple selectors being present on the dashboard

## Related Issue(s)
https://github.com/FlowFuse/node-red-dashboard/issues/882

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: cypress.config.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ module.exports = defineConfig({
         specPattern: 'cypress/tests/**/*.spec.{js,jsx,ts,tsx}',
         setupNodeEvents (on, config) {
             // implement node event listeners here
-        }
+        },
+        defaultCommandTimeout: 10000,
+        retries: { runMode: 3, openMode: 1 }
```
Comment (joepavitt) [inline cypress.config.js:11]: is this required?

Comment (cstns) [reply cypress.config.js:11]: It alleviates some of the flaky tests by retrying them. It's more of a band aid than a permanent fix but could provide some consistency when running e2e tests in CI

image

As can be seen in the logs, some tests that may have failed the first time passed the second or third. The downside is that it somewhat extends the duration of the test runner and accompanying costs

**Comment by arturv2000** [comment]
Was attempting to run the test locally, and stumble upon this.

The best solution seems to separate the UI_Form tests in 3 different dashboard pages, at least locally was having some issues by the fact that there are duplicated elements in dashboard, and the "Dynamic Form" sometimes seems to load the first time with the previous configuration.

Created a branch with the possible fix: [fix_UIForms_Tests](https://github.com/arturv2000/node-red-dashboard/tree/fix_UIForms_Tests)

At least locally they are no longer failing

**Comment by joepavitt** [comment]
Thanks @arturv2000 this is a Pull Request from @cstns so should have the fix included already

**Comment by joepavitt** [comment]
Although, as @cstns mentioned to me on Slack, not sure if this is a "stable" fix, as tests are set to repeat a few times now.

**Pull request merged**